lib/aruba/api.rb

```diff
@@ -1165,69 +1167,36 @@ def with_env(env = {}, &block)
       restore_env
     end
 
+    # Access to announcer
+    def announcer
+      @announcer ||= Announcer.new(
+        self,
+        :stdout => @announce_stdout,
+        :stderr => @announce_stderr,
+        :dir    => @announce_dir,
+        :cmd    => @announce_cmd,
+        :env    => @announce_env
+      )
+
+      @announcer
+    end
+
+    module_function :announcer
+
     # TODO: move some more methods under here!
 
     private
 
     def last_exit_status
```

Comment:
I think will deprecate this method. Users can now use `last_command.exit_status` to get the exit status of the last command executed.